runtime/errors.rs

```diff
@@ -146,6 +155,10 @@ pub fn get_nix_error_class(error: &nix::Error) -> &'static str {
     nix::Error::ENOTTY => "BadResource",
     nix::Error::EPERM => "PermissionDenied",
     nix::Error::ESRCH => "NotFound",
+    nix::Error::ELOOP => "FilesystemLoop",
```

Comment:
Why lowercase `system`?

Reply:
This is mapped directly from `std::io::ErrorKind` which uses `FilesystemLoop`.